airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1723,6 +1723,31 @@ paths:
           $ref: "#/components/responses/NotFoundResponse"
         "422":
           $ref: "#/components/responses/InvalidInputResponse"
+  /v1/source_oauths/complete_oauth_return_secret:
```

Comment:
just curious, why is this a new endpoint instead of a flag on the `complete_oauth` endpoint?

Reply:
I don't have a good reason, happy to turn this into a flag - will work on that.